README.md

```diff
@@ -1 +1,826 @@
-# message-tracing-with-stream-processor
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor.svg?branch=master)](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor)
+
+# Integration with WSO2 Stream Processor
+
+WSO2 stream processor provides us with distributed message tracing. The Distributed Message Tracer allows you to trace the events which are produced while ballerina services serve for requests. The ballerina services send these tracing data as WSO2 events.
```

Comment:
Change lower case to title case for product names so that it reads as "WSO2 Stream Processor" and "Ballerina". Apply everywhere in this guide if applicable.

Also you would normally use "that" instead of "which" in this context. Normally "which" is used to add some additional information to a sentence and you would normally have a comma before "which".

I've also made some other minor modifications here. Shall we use something similar to the following:

"WSO2 Stream Processor (SP) enables you to perform distributed message tracing. The Distributed Message Tracer allows you to trace the events that are produced, while Ballerina services serve the requests. The Ballerina services send the tracing data as WSO2 events to WSO2 SP."

Reply:
Yes sure. I will make the necessary changes.